.github/pull_request_template.md

```diff
@@ -1,18 +1,18 @@
-**Jira:**
+# Jira: [TKT-000](https://clever.atlassian.net/browse/TKT-000)
 
-**Overview:**
+# Overview:
 
-**Screenshots/GIFs:**
+# Screenshots/GIFs:
 
-**Testing:**
+# Testing:
 
 - [ ] Unit tests
 - Manual tests:
   - [ ] Chrome
   - [ ] Safari
   - [ ] IE11
 
-**Roll Out:**
+# Roll Out:
```

Comment:
Just adding a pre-filled Jira ticket link and some more formatting to the section headers.